apps/03TicTacToe.js

```diff
@@ -23,30 +33,139 @@ function printBoard() {
 
 function horizontalWin() {
     // Your code here
+
+    if ((board[0][0] === 'X' && board[0][1] === 'X' && board[0][2] === 'X') || (board[1][0] === 'X' && board[1][1] === 'X' && board[1][2] === 'X') || (board[2][0] === 'X' && board[2][1] === 'X' && board[2][2] === 'X')) {
+        return true;
+    } else if ((board[0][0] === 'O' && board[0][1] === 'O' && board[0][2] === 'O') || (board[1][0] === 'O' && board[1][1] === 'X' && board[1][2] === 'O') || (board[2][0] === 'O' && board[2][1] === 'O' && board[2][2] === 'O')) {
+        return true;
+    } else {
+        return false;
+    }
+
+
 }
 
 function verticalWin() {
     // Your code here
+
+    if ((board[0][0] === 'X' && board[1][0] === 'X' && board[2][0] === 'X') || (board[0][1] === 'X' && board[1][1] === 'X' && board[2][1] === 'X') || (board[0][2] === 'X' && board[1][2] === 'X' && board[2][2] === 'X')) {
```

Comment:
This if and the else if have _very_ similar logic. Maybe we could pull into a local function?!?!?!?

Ask me for deets if you want to go hardcore mode (you better)